src/cmd/root.go

```diff
@@ -25,10 +25,15 @@ and it allows you to perform many common Git operations faster and easier.`,
 
 // Execute runs the Cobra stack.
 func Execute() {
-	if err := git.CheckVersion(); err != nil {
+	majorVersion, minorVersion, err := git.Version()
+	if err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+	if majorVersion*100+minorVersion < 207 {
```

Comment:
This feels weird to me. Thoughts on `majorVersion > 2 || (majorVersion == 2 and minorVersion > 7)`?

Reply:
Great idea, that's what I was looking for. Implemented in #1547. I'm checking a bunch of edge cases (2.6, 3.0, etc) using unit tests so that we still only have one end-to-end test.